packages/yoroi-extension/app/ergo-connector/components/connect/ConnectPage.scss

```diff
@@ -1,5 +1,6 @@
 .component {
   height: calc(100vh - 92px);
+  overflow: scroll;
```

Comment:
Please change to `auto`, otherwise, there is a white square:
![bottom-right](https://user-images.githubusercontent.com/1028789/155266193-4398a1f4-aed3-4296-98c4-18b0cb73a2a3.png)

Alternatively, change the color of the background to match so that there is no white squre.